src/game-logic/reducers/computeStateForNextDay.js

```diff
@@ -36,8 +38,8 @@ const adjustItemValues = state => ({
  * @param {farmhand.state} state
  * @returns {farmhand.state}
  */
-export const computeStateForNextDay = (state, isFirstDay = false) =>
-  (isFirstDay
```

Comment:
haha it's all good!

Reply:
I'm glad you're not too bothered by it! 😅

I was developing Farmhand solo for a long time at first, and I assumed that would always be the case (though I'm happy that I was wrong!). I wanted to try out a few coding ideas because of that (many of which are still present on the code base today). Some worked well, but some didn't. 😆